File: foundation/foundation 3 month/lesson8/homework/ex_9.py
def check(x):
	if x[0]=='+' and x[1]=='9' and x[2]=='9' and x[3]=='8':
		return True

k=int(input("Raqamlar soni: "))
arr=[]
for x in range(1,k+1):
	num=input(f"{x} chi raqamni kiriting: ")
	arr.append(num)
print(arr)
ls=list(filter(check,arr))
print(ls)

File: foundation/foundation 2 month/lesson8/homework/ex_9.c
#include <stdio.h>

int check(int k)
{
	if(k<10)
	return k ;
	return k%10 +check(k/10);
}
int main()
{
	int m;
	printf("N ni kiriting: ");
	scanf("%d",&m);
	int d=check(m);
	printf("%d",d);
}

